### PR TITLE
bump: Add the roadmap for v1.6.0

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: Denis Tingaikin <denis.tingajkin@xored.com>

### CODE REVIEW

File: content/docs/roadmap/v1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+### Public cluster testing via clusterctl
+
+This makes our builds consistent and repeatable  across a wide variety of infrastructure environments. 
+
+Planned to migrate next repos to the `clusterctl`
+
+- integration-k8s-aws
+- integration-k8s-gke
+- integration-k8s-aks
```
Comment (edwarnicke) [inline content/docs/roadmap/v1.6.0.md:76]: What about clusterctl and integration-k8s-packet?

File: content/docs/roadmap/v1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+- Open Polic Agent
+- OpenTelemetry
+- gRPC
+- protobuf
+- JWT
+
+To keep NSM uptodate.
+
+
```
Comment (edwarnicke) [inline content/docs/roadmap/v1.6.0.md:32]: This is definitely a thing to do for v1.6, but let's keep the roadmap to new software features/improvements. :)

File: content/docs/roadmap/v1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+### Improve stability and bug fixes
+
+Planned to resolve all reported issues with milestone `v1.6.0`.
+And and all new stability issues that will be reported before the release!
+
```
Comment (edwarnicke) [inline content/docs/roadmap/v1.6.0.md:37]: This is definitely a thing to do for v1.6, but let's keep the roadmap to new software features/improvements. :)

File: content/docs/roadmap/v1.6.0.md (hunk 1 above)
Comment (denis-tingaikin) [reply content/docs/roadmap/v1.6.0.md:76]: Oops!. Fixed.

File: content/docs/roadmap/v1.6.0.md (hunk 2 above)
Comment (denis-tingaikin) [reply content/docs/roadmap/v1.6.0.md]: Sure. Fixed.

File: content/docs/roadmap/v1.6.0.md (hunk 3 above)
Comment (denis-tingaikin) [reply content/docs/roadmap/v1.6.0.md]: Sure. Fixed.

**Review by edwarnicke — APPROVED** [review]

**Pull request merged**